### PR TITLE
Added logic to get switch, routes and neighbors table

### PR DESCRIPTION
**Pull request by sbhagwatvnera**

### CODE REVIEW

File: network_insight_sdk_generic_datasources/common/physical_device.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def execute_commands(self):
                     command_output_dict[workload[COMMAND_KEY]] = command_result
                     py_logger.info('Command %s Result %s' % (workload[COMMAND_KEY], command_result))
                     table = self.parse_command_output(workload, command_result)
-                if 'switch' in command_id:
```
Comment (sbhagwatvnera) [inline network_insight_sdk_generic_datasources/common/physical_device.py]: This was updating the switch-port table as well.

File: network_insight_sdk_generic_datasources/routers_and_switches/dell/dell_pre_post_processor.py [hunk 2]
```diff
@@ -102,8 +129,93 @@ def post_process(self, data):
                         rv = v.replace('(', '').replace(')', '')
                         result_vlans.append(str(rv))
                     elif '-' in v:
-                        ran = v.split('-')
-                        rv = range(int(ran[0]), int(ran[1]) + 1)
-                        result_vlans = result_vlans + rv
+                        result_vlans = result_vlans + v.split('-')
```
Comment (piyushvmware) [inline network_insight_sdk_generic_datasources/routers_and_switches/dell/dell_pre_post_processor.py]: Put a comment that we need to handle this product.

File: network_insight_sdk_generic_datasources/routers_and_switches/dell/dell_pre_post_processor.py [hunk 3]
```diff
@@ -54,14 +74,21 @@ def post_process(self, data):
...
+        :param data: show interfaces command output
+        :return: List of dict of switch ports
+        """
         result = []
         for d in data:
             if 'accessVlan' in d:
                 d['accessVlan'] = ''  # TODO
             if 'interfaceSpeed' in d:
-                d['interfaceSpeed'] = str(int(d['interfaceSpeed']) * 1000000)
+                d['interfaceSpeed'] = '0' #str(int(d['interfaceSpeed']) * 1000000)
```
Comment (piyushvmware) [inline network_insight_sdk_generic_datasources/routers_and_switches/dell/dell_pre_post_processor.py]: Put a comment that we need to handle this in product.

Comment (piyushvmware) [reply network_insight_sdk_generic_datasources/routers_and_switches/dell/dell_pre_post_processor.py]: Like 
\# TODO Handle in product.

**Pull request merged**